Makefile

```diff
@@ -0,0 +1,9 @@
+restore:
```

Comment:
ok. I always seem to have problems with godep, can you tell me what I
should be doing here? I'd like to automate the process of updating
dependencies.

On Mon, 2015-01-05 at 18:15 -0800, Alexander Morozov wrote:
> In Makefile:
> 
> > @@ -0,0 +1,9 @@
> > +restore:
> 
> Restore shouldn't be used at all. We vendor all dependencies
> 
> —
> Reply to this email directly or view it on GitHub.
> 
>